scriptmodules/emulators/ppsspp.sh

```diff
@@ -157,9 +164,9 @@ function configure_ppsspp() {
     ln -snf "$romdir/psp" "$md_conf_root/psp/PSP/GAME"
 
     if isPlatform "tinker"; then
-        addEmulator 0 "$md_id" "psp" "$md_inst/PPSSPPSDL --fullscreen %ROM%"
+        addEmulator 1 "$md_id" "psp" "$md_inst/PPSSPPSDL --fullscreen %ROM%"
     else
-        addEmulator 0 "$md_id" "psp" "$md_inst/PPSSPPSDL %ROM%"
+        addEmulator 1 "$md_id" "psp" "$md_inst/PPSSPPSDL %ROM%"
```

Comment:
Sorry just noticed this as I was about to merge - Not sure this is relevant to the PR ? Changing the default emulator for ppsspp.

Reply:
Yes good spot, thank you.
I couldn't access the runcommand popup to select it back when I was runtime testing (starting emulationstation via a mobile SSH app) so it wouldn't start.  We can't use the libretro plugin at the moment, but that's no need to make the standalone default here.
Fixed -> commited -> squashed.